tests/densify/test_mvs_patchmatchnet.py

```diff
@@ -0,0 +1,120 @@
+"""Unit tests for using Patchmatchnet method for dense point cloud reconstruction
+
+In this unit test, we make a simple scenario that eight cameras are around a circle (radius = 40.0).
+Every camera's pose is towards the center of the circle (0, 0, 0).
+
+There are 30 toy points uniformly located at the same line:
+  /  z - 2y = 0
```

Comment:
i prefer the syntax `ax + by + cz + d = 0`. that's what we use elsewhere. So maybe we could say
```python
#     0x + -2y + 1z = 0, i.e. x=0
```